### PR TITLE
Sign up page updates

### PR DESCRIPTION
**Pull request by jenhadfield-dfe**
### Trello card
https://trello.com/c/UD9VZ0FI/6259-carry-out-a-final-check-of-expired-images

### CODE REVIEW

File: app/webpacker/images/content/mailing-list-signup/left.jpg
Comment (mariosh-e) [inline app/webpacker/images/content/mailing-list-signup/left.jpg:1]: All the new images look grainy/pixelated on my screen?

**Review by mariosh-e — COMMENTED** [review]
Looks better removing the mobile image however the images on both pages don't look at clear anymore, something worth double checking?

**Review by mariosh-e — APPROVED** [review]
Looks all good to me. The images look clearer

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Pull request merged**